source/geh_calculated_measurements/src/geh_calculated_measurements/database_migrations/migrations_runner.py

```diff
@@ -10,19 +12,17 @@
 from geh_calculated_measurements.database_migrations.settings.catalog_settings import CatalogSettings
 
 
+@start_trace()
 def migrate() -> None:
-    spark_sql_migrations_configuration = _configure_spark_sql_migrations()
-    migration_pipeline.migrate(spark_sql_migrations_configuration)
-
+    log = Logger(__name__)
+    catalog_settings = CatalogSettings()
+    log.info(f"Initializing migrations with CatalogSettings: {catalog_settings}")
 
-def _configure_spark_sql_migrations() -> SparkSqlMigrationsConfiguration:
-    substitution_variables = substitutions.substitutions()
-    catalog_name = CatalogSettings().catalog_name  # type: ignore
-
-    return SparkSqlMigrationsConfiguration(
+    spark_sql_migrations_configuration = SparkSqlMigrationsConfiguration(
         migration_schema_name=MeasurementsCalculatedInternalDatabaseDefinition.measurements_calculated_internal_database,
         migration_table_name=MeasurementsCalculatedInternalDatabaseDefinition.executed_migrations_table_name,
         migration_scripts_folder_path="geh_calculated_measurements.database_migrations.migration_scripts",
-        substitution_variables=substitution_variables,
-        catalog_name=catalog_name,
+        substitution_variables=substitutions.substitutions(),
+        catalog_name=catalog_settings.catalog_name,
```

Comment:
nit (but mentioned because you did it on the line above)
```suggestion
        catalog_name=CatalogSettings().catalog_name,
```

Reply:
Improved comments instead :)